lib/td/client.rb

```diff
@@ -89,8 +89,8 @@ def database(db_name)
   end
 
   # @return [true]
-  def create_log_table(db_name, table_name)
-    @api.create_log_table(db_name, table_name)
+  def create_log_table(db_name, table_name, params)
+    @api.create_log_table(db_name, table_name, params)
```

Comment:
These 2 lines change are necessary?
Looks no code is using added `params` argument.
And I'm worrying about this change breaks here: https://github.com/treasure-data/td-client-ruby/blob/50631e6f9e4fb8615ef186689a3a9b35e0367cd4/lib/td/client/model.rb#L98-L102